### ISSUE TITLE
NaN While Training

### ISSUE THREAD

**Issue by lachjames** [NVIDIA/flowtron#40]
Hi :)

I'm trying to get flowtron working but I've been having this persistent issue where I get nan errors:

*** Stack Trace ***
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
... (this repeats many times) ...
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
Mean None
LogVar None
Prob None
Validation loss 0:       nan
WARNING:root:NaN or Inf found in input tensor.
*** End Stack Trace ***

I've had this error on both my personal Windows machine, and a cloud Linux machine. I'm wondering whether there is some sort of Pytorch version compatibility issue or something?

I'd be very grateful for any assistance that could be provided!

**Comment by lachjames** [comment]
> replace .byte() with .bool()
> https://github.com/NVIDIA/flowtron/blob/master/flowtron.py#L33

This seems to have fixed the problem, but I'm still curious as to why it did so.

**Comment by rafaelvalle** [comment]
It's an issue with pytorch versions.
